### ISSUE TITLE
New accounts can't buy first ship with new url routes

### ISSUE THREAD

**Issue by kurt1288** [SpaceTradersAPI/issues-and-suggestions#122]
This is only regarding the new API routes. The old routes do not have this issue (as the old routes expose a systems endpoint).

For new accounts, there is no way to get systems since there is no endpoint in the new API that returns systems. Without knowing any systems, you can't find locations, and without locations you can't buy a ship (and without a ship you can't know any locations).

**Comment by nhowell** [comment]
I can't upvote this enough. It's a bummer that this is still a problem with the new API endpoints. I want to move my UI over to the new endpoints, but essentially can't due to this issue.

**Comment by towerofpower256** [comment]
Yep, completely agree. I'm just starting out, and I was wracking my brain trying to figure out how to buy a ship in the first place without knowing any system symbols.

Possible solution: what if there was an endpoint to list all systems known to the player, and new accounts start with a small cluster of them "revealed" to them?

**Comment by space-admiral** [comment]
Sorry we haven't tackled this yet - this will be solved in the V2 API coming out soon. The first system should be hard coded in your client (OE) which we hard code in the getting started guide. In the new API you will be able to view all systems without a ship. Let me know if that helps!

**Comment by towerofpower256** [comment]
In the mean time, the undocumented endpoint `/game/systems` gets all systems & locations.
